ecommerce_api_client/tests/test_auth.py

```diff
@@ -43,7 +62,23 @@ def test_headers(self):
     @httpretty.activate
     def test_tracking_context(self):
         """ Verify the tracking context is enclosed in the token payload, when specified. """
-        self.assert_expected_token_value({'foo': 'bar'})
+        self.assert_expected_token_value(tracking_context={'foo': 'bar'})
+
+    @httpretty.activate
+    def test_issuer(self):
+        """ Verify that the issuer is enclosed in the token payload, when specified. """
+        self.assert_expected_token_value(issuer='http://example.com/oauth')
+
+    @httpretty.activate
+    @patch.object(JwtAuth, 'datetime_now', Mock(return_value=CURRENT_TIME))
+    def test_expires_in(self):
+        """ Verify the expiration date is enclosed in the token payload, when specified. """
+        self.assert_expected_token_value(expires_in=60)
+
+    def test_datetime_now(self):
+        """ Validate that the datetime_now returns a valid datetime object """
+        auth = JwtAuth(self.username, self.full_name, self.email, self.signing_key)
+        self.assertIsInstance(auth.datetime_now(), datetime.datetime)
```

Comment:
@jimabramson @clintonb I agree. However, I had an extensive conversation with @nedbat yesterday, and after much discussion we found that this test was the only way to ensure 100% test coverage. Without it, coveralls reports an error stating that the test coverage dropped from 100% to 97.18%.